aospy/calc.py

```diff
@@ -601,8 +602,7 @@ def _save_files(self, data, dtype_out_time):
                 reg_data = xr.open_dataset(path)
             except (EOFError, RuntimeError, IOError):
                 reg_data = xr.Dataset()
-            # Add the new data to the dictionary or Dataset.
-            # Same method works for both.
+            # Add the new data to the Dataset.
```

Comment:
Actually, this logic is now self-explanatory, so I'd just omit the comment altogether.